src/geometry/angle.rs

```diff
@@ -7,6 +7,7 @@ use float_cmp::{ApproxEq, F32Margin};
 #[allow(unused_imports)]
 use micromath::F32Ext;
 
+#[allow(dead_code)]
```

Comment:
Perhaps a bit too nitpicky, but maybe this would be better on the `const ANGLE_90DEG` as that's the only item that triggers a warning.

Reply:
I would never be so nitpicky... Oh wait, I would, never mind :wink: I've applied your suggestion.

Reply:
I do learn from the best to be fair. I noticed you had this attribute on the `const`s themselves in other places too, so I feel vindicated in making this comment :grin: